lib/image.rb

```diff
@@ -44,6 +47,9 @@ class Image
     # it can be placed here
     attr_accessor :intrinsics
 
+    # Preview image blob
+    attr_accessor :preview
```

Comment:
Offset and byte size of preview image depends on the size and seek limits that is set in the FormatParser class. If its offset and size is bigger than the limits, preview returns nil. Would you still like me to remove it?

Reply:
To be very honest yes, primarily due to the intended use of the library. It is made to scan a lot of files very quickly, and we want to get information that will be useful for display but no actual pixel/audio data, since it will end up discarded anyway - we use other tools (ElasticTranscoder, vips and ImageMagick) to do the actual processing. If we return this data to the caller it will take longer to garbage collect it, it will allocate a Ruby string for each preview we pass, and all of the above won't end up being used by the calling code. So if we want to return that information in some useful form IMO we should include the byte offset at which to find the preview blob and it's size in the intrinsics, but not the actual binary string with the preview.

Reply:
Oh, now I see what you mean. I will update the class accordingly.